observability/logging/logging-6.1/log6x-release-notes-6.1.adoc

```diff
@@ -0,0 +1,81 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-release-notes-6.1"]
+= Logging 6.0.0
```

Comment:
should the title be Logging 6.1.0?

Reply:
We will incorporate/make updates after https://github.com/openshift/openshift-docs/pull/84666 is merged.

Reply:
https://issues.redhat.com/browse/OBSDOCS-1494

Reply:
....yikes.

Reply:
There is absolutely no reason or justification for not correcting the title of an assembly before its merged.